src/tasks2/task1.js

```diff
@@ -0,0 +1,19 @@
+// TODO: 1. Написать функцию, которая принимает массив чисел. Функция должна вернуть массив, в котором все элементы
+// принятого массива умножены на 3;
+
+
+const multiplyArrayElements = (numericArray) => {
+    if (Array.isArray(numericArray)) {
+        if (numericArray.every(elem => typeof elem === "number")) {
+            numericArray.forEach(((element, index) => numericArray[index] = 3 * element));
```

Comment:
1)Your entire array will be mutated. When you wok with non-primitive data types, you have to do the copy of it, and then work with the copy.
2)For this task the better solution is to use .map() method